README.md

```diff
@@ -1,24 +1,44 @@
-# README
+# OOP-School-Library
+- This is a school library that uses OOP to build record app for adding students, teachers, book rentals and list all the available rented books alongside either the teacher or student and indicates the date. The app organizes everything in a great way allowing easy maneuver.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Built With
+- Languages: Ruby on Rails
+- Models/Frameworks: OOP
```

Comment:
- [ ] Kindly update your readme title and description to match the project.

- [ ] There should be `config/database.yml` file that defines the database adapter. And the adapter should use PostgreSQL. This file also defines the databases for different environments.

  Something like this: [My config/database file](https://github.com/julie-ify/Blog-app/blob/main/config/database.yml)

Reply:
Hi @julie-ify , thank you for the suggestions. I have implemented the changes.